pkg/controller/kappnav/kappnav_controller.go

```diff
@@ -168,40 +199,64 @@ func (r *ReconcileKappnav) Reconcile(request reconcile.Request) (reconcile.Resul
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
-
+	
 	// Call factory method to create new KappnavExtension
 	extension := kappnavutils.NewKappnavExtension()
 
 	// Apply defaults to the Kappnav instance
 	err = kappnavutils.SetKappnavDefaults(instance)
+	if (logger.IsEnabled(kappnavutils.LogTypeInfo)) {
+		logger.Log(kappnavutils.CallerName(), kappnavutils.LogTypeInfo, "Apply defaults to kappnav instance" + otherLogData, logName)
+	}
 	if err != nil {
-		reqLogger.Error(err, "Failed to process default values file")
+		if (logger.IsEnabled(kappnavutils.LogTypeError)) {
+			logger.Log(kappnavutils.CallerName(), kappnavutils.LogTypeError, fmt.Sprintf("Failed to process default values file" + otherLogData + ", Error: %s", err), logName)
+		}
 		return reconcile.Result{}, err
 	}
 
+	// Retrieve logging info from kappnav CR and update the kappnavutils level	
+	if (logger.IsEnabled(kappnavutils.LogTypeInfo)) {
+		logger.Log(kappnavutils.CallerName(), kappnavutils.LogTypeInfo, "Retrieve kappnavutils info from kappnav CR" + otherLogData, logName)
+	} 	
+	loggingMap := instance.Spec.Logging	
+	for key, value := range loggingMap {	
```

Comment:
This doesn't need to be a loop. You can access the value from a map directly. loggingMap["operator"].

Reply:
It was done in the earlier change and forgot to change it after other changes.